app/utils/graphql.ts

```diff
@@ -48,24 +46,8 @@ const fetcher = <T>([query, variables, token, hasuraRoleName, schema]: [
       Authorization: `Bearer ${token}`,
       "x-hasura-role": hasuraRoleName,
     },
-  }).then((data) => {
-    return schema.parseAsync(data);
```

Comment:
👋 goodbye runtime validation. I dunno! now i kind of miss that we know 100% that are types match our graphql queries. Last chance to change our minds! ;)

Reply:
what would be the cause of our types not matching our graphql queries?

Reply:
if we changed a field name without updating our types, or if we add a field to a graphql query without adding it to our type. and vice versa. just these marginal edge cases.

Reply:
in the same way that it's our responsibility to update the Hasura metadata when we add a column to a db table, we also need to make sure we add the new column to our queries and our types.

Reply:
I created https://github.com/cityofaustin/atd-data-tech/issues/20272 to look into the code-generated types—and i think if we get to that point where we know 100% that our types match our API we should be able to rest pretty easy. Will be fun to pursue this!